securethenews/whitenoise_storage.py

```diff
@@ -0,0 +1,5 @@
+from whitenoise.storage import CompressedManifestStaticFilesStorage
+
+
+class WhiteNoiseStaticFilesStorage(CompressedManifestStaticFilesStorage):
+    manifest_strict = False
```

Comment:
Nitpick: I think it would feel more Django idiomatic to just call this file `storage.py`